.travis.yml

```diff
@@ -1,24 +1,22 @@
-sudo: false
+os: linux
+dist: xenial
```

Comment:
is there a reason to specify the release?

Reply:
We don't have to, but we get a warning that it is missing if we don't. It might be worth the warning just so that it floats to their default.

Reply:
I see, how about using `bionic`?